src/CoinParamUtils.cpp

```diff
@@ -129,6 +130,129 @@ std::string nextField(const char *prompt)
 
 namespace CoinParamUtils {
 
+std::string printString(const std::string input, int maxWidth)
+{
+   std::ostringstream buffer;
+   std::istringstream iss(input);
+   int currentWidth = 0;
+   std::string word;
+   while (iss >> word){
+      currentWidth += word.length();
+      if (currentWidth >= 65){
+         buffer << std::endl << word;
```

Comment:
```
currentWidth = 0; 
```
otherwise after the first full line a line break will be put before all following words

Reply:
Yes, thanks!